### PR TITLE
Add is_modular(L, p) for ZZLat

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

**Comment by thofma** [comment]
@StevellM please give this a quick glance

File: test/QuadForm/Quad/ZLattices.jl [hunk 1]
```diff
@@ -712,6 +712,7 @@ let
   @test Hecke.init(C) isa Hecke.ZLatAutoCtx
 end
 
+<<<<<<< HEAD
```
Comment (StevellM) [inline test/QuadForm/Quad/ZLattices.jl:715]:
```suggestion
```

**Comment by StevellM** [comment]
It looks good. Maybe could you duplicate the documentation ? The global documentation will not cover this function since the input is not an ideal but just an integer.

**Comment by thofma** [comment]
There is already a docstring for `is_modular(::AbstractLat, p)`. I adjusted the wording.

**Pull request merged**